components/applications-service/pkg/storage/postgres/service_groups.go

```diff
@@ -5,6 +5,7 @@ import (
 	"strings"
 
 	"github.com/chef/automate/components/applications-service/pkg/storage"
+	"github.com/chef/automate/components/secrets-service/utils"
```

Comment:
I would suggest that, instead of depending on a component's code, we should move this code to a library. I have verified and there is more than one place where we define and use these functions so, it makes more sense for us to maintain a single codebase.  

https://github.com/chef/automate/search?q=IsSqlSafe&unscoped_q=IsSqlSafe

Reply:
Agree 100%. We already noticed this and are planning to do it in a separate PR so we don't have features for one team mixed with a refactor that affects a lot of teams in one patch.

Reply:
I agree.  We should avoid cross-component dependencies if we don't need them.  I'd say either copy the code or put it in lib.  I think there is maybe a good argument for a pgutils.  Some other functions that might be helpful there:

https://github.com/chef/automate/blob/master/lib/platform/pg/queries.go#L91-L129
https://github.com/chef/automate/blob/master/lib/cereal/postgres/postgres.go#L890-L902

and probably a number of others around the code base.